images/go-ipfs/tests/docker-test.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+CONTAINER_NAME="ipfs-host-$(uuidgen)"
+CONTAINER_PORT=${FREE_PORT}
+REQUEST_RETRIES=5
+RETRY_DELAY=15
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# Log entries which we expect to be present when the service is running healthy.
+expected_logs=(
+  "Daemon is ready"
+)
+
+missing_logs=()
+
+# Start up a new IPFS container
+TEST_start_container() {
+  container_id=$(docker run \
+    -d --rm \
+    --name "${CONTAINER_NAME}" -p "${CONTAINER_PORT}:5001" \
+    ipfs/kubo:latest)
```

Comment:
```suggestion
    ${IMAGE_NAME})
```

This test needs to run using the image that's being built, not the one from docker hub.